docs/reference/ml/df-analytics/apis/dfanalyticsresources.asciidoc

```diff
@@ -124,4 +123,117 @@ An {oldetection} configuration object has the following properties:
 `feature_influence_threshold`:: 
   (double) The minimum {olscore} that a document needs to have in order to 
   calculate its {fiscore}. 
-  Value range: 0-1 (`0.1` by default).
\ No newline at end of file
+  Value range: 0-1 (`0.1` by default).
+  
+
+[discrete]
+[[regression-resources]]
+==== {regression-cap} configuration objects
+
+A `{regression}` configuration object has the following properties:
+
+`dependent_variable`::
+  (Required, string) Defines which field of the {dataframe} is to be predicted. 
+  This parameter is supplied by field name and must match one of the fields in 
+  the index being used to train. If this field is missing from a document that 
+  row will not be used for training, but a prediction with the trained model 
+  will be generated for it. The data type of the field must be numeric. It is 
+  also known as continuous target variable.  
+
+`eta`::
+  (Optional, double) The shrinkage applied to the weights. Smaller values result 
+  in larger forests which have better generalization error. However, the smaller 
+  the value the longer the training will take and the inference will also be 
+  larger and slower. For more information, see 
+  https://en.wikipedia.org/wiki/Gradient_boosting#Shrinkage[this wiki article] 
+  about shrinkage.
+
+`feature_bag_fraction`::
+  (Optional, double) Defines the fraction of features which will be used when 
+  selecting a random bag for each candidate split.
+  
+`maximum_number_trees`::
+  (Optional, integer) Defines the maximum number of trees the forest is 
+  allowed to contain. The maximum value is 2000.
+    
+`prediction_field_name`::
+  (Optional, string) Defines the name of the prediction field in the results. 
+  Defaults to `<dependent_variable>_prediction`.
+    
+`training_percent`::
+  (Optional, integer) Defines what percentage of the data rows that are 
+  eligible for training. Defaults to `100`.
+
+
+===== {regression-cap} regularization parameters
+
+Regularization parameters are constant multipliers of a regularization term 
+added to the optimization objective. They aim to prevent overfitting on the 
+training data set and improve generalization performance. These parameters are 
+calculated and set automatically by 
+<<ml-hyperparameter-optimization,hyperparameter optimization>> if you don't 
+supply their values manually. For more 
```

Comment:
Note that `eta` and `feature_bag_fraction` are also calculated automatically to give the best validation errors. I sort of feel like it would be nice to capture this point somewhere which makes this clear.